[2024-04-23T12:10:30.851+0000] {processor.py:161} INFO - Started process (PID=51) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:10:30.862+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:10:30.889+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:10:30.888+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:10:31.066+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:10:31.092+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:10:31.366+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:10:31.841+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:10:32.426+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:10:32.425+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:10:32.746+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:10:32.746+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:10:35.028+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 4.246 seconds
[2024-04-23T12:11:05.893+0000] {processor.py:161} INFO - Started process (PID=65) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:11:06.268+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:11:06.274+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:11:06.273+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:11:06.439+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:11:06.440+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:11:06.441+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:11:06.458+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:11:06.516+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:11:06.515+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:11:06.587+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:11:06.586+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:11:08.957+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 3.071 seconds
[2024-04-23T12:11:44.677+0000] {processor.py:161} INFO - Started process (PID=83) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:11:44.691+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:11:44.700+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:11:44.699+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:11:44.741+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:11:44.743+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:11:44.744+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:11:44.765+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:11:44.835+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:11:44.835+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:11:44.890+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:11:44.889+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:11:45.760+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.100 seconds
[2024-04-23T12:12:17.656+0000] {processor.py:161} INFO - Started process (PID=98) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:12:17.674+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:12:17.678+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:12:17.677+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:12:18.038+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:12:18.039+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:12:18.058+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:12:18.161+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:12:18.628+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:12:18.627+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:12:18.848+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:12:18.848+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:12:19.096+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.541 seconds
[2024-04-23T12:12:51.655+0000] {processor.py:161} INFO - Started process (PID=113) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:12:51.658+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:12:51.663+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:12:51.662+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:12:51.713+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:12:51.726+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:12:51.727+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:12:51.786+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:12:52.655+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:12:52.655+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:12:52.758+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:12:52.757+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:12:52.866+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.244 seconds
[2024-04-23T12:13:23.653+0000] {processor.py:161} INFO - Started process (PID=129) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:13:23.657+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:13:23.663+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:13:23.661+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:13:23.719+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:13:23.720+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:13:23.721+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:13:23.751+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:13:23.828+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:13:23.828+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:13:23.927+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:13:23.926+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:13:24.177+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.541 seconds
[2024-04-23T12:13:54.847+0000] {processor.py:161} INFO - Started process (PID=146) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:13:54.849+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:13:54.853+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:13:54.852+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:13:55.002+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:13:55.003+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:13:55.005+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:13:55.026+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:13:55.084+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:13:55.084+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:13:55.131+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:13:55.131+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:13:56.044+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.208 seconds
[2024-04-23T12:14:26.656+0000] {processor.py:161} INFO - Started process (PID=159) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:14:26.677+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:14:26.685+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:14:26.684+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:14:27.338+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:14:27.340+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:14:27.340+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:14:27.379+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:14:27.493+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:14:27.492+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:14:27.643+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:14:27.643+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:14:27.833+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.214 seconds
[2024-04-23T12:14:58.254+0000] {processor.py:161} INFO - Started process (PID=176) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:14:58.256+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:14:58.261+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:14:58.261+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:14:58.318+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:14:58.319+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:14:58.320+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:14:58.347+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:14:58.434+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:14:58.428+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:14:58.522+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:14:58.521+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:15:00.020+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.797 seconds
[2024-04-23T12:15:30.176+0000] {processor.py:161} INFO - Started process (PID=194) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:15:30.179+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:15:30.183+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:15:30.182+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:15:30.214+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:15:30.215+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:15:30.216+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:15:30.235+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:15:30.280+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:15:30.279+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:15:30.316+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:15:30.315+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:15:30.357+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.188 seconds
[2024-04-23T12:16:00.881+0000] {processor.py:161} INFO - Started process (PID=211) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:16:00.884+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:16:00.888+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:16:00.887+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:16:00.934+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:16:00.935+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:16:00.938+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:16:00.957+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:16:01.042+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:16:01.035+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:16:01.150+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:16:01.149+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:16:01.264+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.396 seconds
[2024-04-23T12:16:31.907+0000] {processor.py:161} INFO - Started process (PID=233) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:16:31.909+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:16:31.912+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:16:31.911+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:16:31.945+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:16:31.956+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:16:31.957+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:16:31.973+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:16:32.010+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:16:32.009+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:16:32.051+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:16:32.050+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:16:32.082+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.187 seconds
[2024-04-23T12:17:02.444+0000] {processor.py:161} INFO - Started process (PID=249) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:17:02.446+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:17:02.451+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:17:02.450+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:17:02.492+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:17:02.493+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:17:02.493+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:17:02.515+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:17:02.565+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:17:02.565+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:17:02.622+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:17:02.621+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:17:02.671+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.243 seconds
[2024-04-23T12:17:32.784+0000] {processor.py:161} INFO - Started process (PID=264) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:17:32.785+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:17:32.788+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:17:32.787+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:17:32.815+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:17:32.816+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:17:32.817+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:17:32.828+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:17:32.870+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:17:32.870+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:17:32.912+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:17:32.911+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:17:32.948+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.175 seconds
[2024-04-23T12:18:03.039+0000] {processor.py:161} INFO - Started process (PID=281) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:18:03.041+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:18:03.045+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:18:03.044+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:18:03.073+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:18:03.074+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:18:03.075+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:18:03.088+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:18:03.126+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:18:03.125+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:18:03.164+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:18:03.164+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:18:03.207+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.181 seconds
[2024-04-23T12:18:33.335+0000] {processor.py:161} INFO - Started process (PID=297) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:18:33.336+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:18:33.340+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:18:33.340+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:18:33.372+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:18:33.382+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:18:33.386+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:18:33.449+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:18:33.537+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:18:33.536+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:18:33.656+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:18:33.656+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:18:33.754+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.430 seconds
[2024-04-23T12:19:04.564+0000] {processor.py:161} INFO - Started process (PID=313) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:19:04.566+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:19:04.571+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:19:04.569+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:19:04.603+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:19:04.603+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:19:04.604+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:19:04.620+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:19:04.669+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:19:04.669+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:19:04.726+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:19:04.725+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:19:04.771+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.217 seconds
[2024-04-23T12:19:35.308+0000] {processor.py:161} INFO - Started process (PID=330) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:19:35.310+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:19:35.314+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:19:35.313+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:19:35.351+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:19:35.352+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:19:35.353+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:19:35.374+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:19:35.426+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:19:35.425+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:19:35.482+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:19:35.481+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:19:35.520+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.224 seconds
[2024-04-23T12:20:06.543+0000] {processor.py:161} INFO - Started process (PID=346) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:20:06.545+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:20:06.549+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:20:06.548+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:20:06.577+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:20:06.578+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:20:06.579+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:20:06.592+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:20:06.632+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:20:06.632+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:20:06.671+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:20:06.670+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:20:06.708+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.176 seconds
[2024-04-23T12:20:36.890+0000] {processor.py:161} INFO - Started process (PID=361) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:20:36.891+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:20:36.895+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:20:36.894+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:20:36.923+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:20:36.924+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:20:36.925+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:20:36.943+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:20:36.979+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:20:36.979+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:20:37.022+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:20:37.022+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:20:37.064+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.185 seconds
[2024-04-23T12:21:07.403+0000] {processor.py:161} INFO - Started process (PID=377) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:21:07.405+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:21:07.414+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:21:07.413+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:21:07.450+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:21:07.451+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:21:07.452+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:21:07.473+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:21:07.546+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:21:07.545+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:21:07.617+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:21:07.617+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:21:07.708+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.318 seconds
[2024-04-23T12:21:38.436+0000] {processor.py:161} INFO - Started process (PID=393) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:21:38.438+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:21:38.462+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:21:38.453+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:21:38.500+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:21:38.501+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:21:38.502+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:21:38.528+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:21:38.907+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:21:38.906+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:21:38.983+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:21:38.983+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:21:40.095+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.675 seconds
[2024-04-23T12:22:10.584+0000] {processor.py:161} INFO - Started process (PID=409) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:22:10.586+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:22:10.590+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:22:10.590+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:22:10.616+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:22:10.617+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:22:10.618+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:22:10.638+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:22:10.700+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:22:10.699+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:22:10.805+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:22:10.804+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:22:10.874+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.299 seconds
[2024-04-23T12:22:41.030+0000] {processor.py:161} INFO - Started process (PID=425) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:22:41.031+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:22:41.035+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:22:41.034+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:22:41.062+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:22:41.063+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:22:41.063+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:22:41.078+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:22:41.116+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:22:41.115+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:22:41.191+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:22:41.187+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:22:41.271+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.251 seconds
[2024-04-23T12:23:14.786+0000] {processor.py:161} INFO - Started process (PID=441) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:23:14.789+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:23:14.794+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:23:14.793+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:23:14.844+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:23:14.844+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:23:14.845+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:23:14.868+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:23:14.930+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:23:14.930+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:23:14.976+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:23:14.976+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:23:15.022+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.258 seconds
[2024-04-23T12:23:49.026+0000] {processor.py:161} INFO - Started process (PID=457) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:23:49.028+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:23:49.032+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:23:49.031+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:23:49.075+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:23:49.076+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:23:49.077+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:23:49.097+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:23:49.208+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:23:49.207+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:23:49.341+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:23:49.341+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:23:49.449+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.440 seconds
[2024-04-23T12:24:20.513+0000] {processor.py:161} INFO - Started process (PID=472) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:24:20.515+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:24:20.520+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:24:20.519+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:24:20.571+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:24:20.575+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:24:20.576+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:24:20.603+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:24:20.689+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:24:20.688+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:24:20.743+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:24:20.743+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:24:20.778+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.286 seconds
[2024-04-23T12:24:56.134+0000] {processor.py:161} INFO - Started process (PID=488) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:24:56.136+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:24:56.145+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:24:56.144+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:24:56.188+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:24:56.190+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:24:56.194+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:24:56.216+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:24:56.272+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:24:56.272+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:24:56.332+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:24:56.331+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:24:56.380+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.267 seconds
[2024-04-23T12:25:36.151+0000] {processor.py:161} INFO - Started process (PID=514) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:25:36.153+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:25:36.156+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:25:36.156+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:25:36.184+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:25:36.185+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:25:36.185+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:25:36.204+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:25:36.250+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:25:36.250+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:25:36.306+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:25:36.306+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:25:36.350+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.210 seconds
[2024-04-23T12:26:09.305+0000] {processor.py:161} INFO - Started process (PID=528) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:26:09.308+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:26:09.313+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:26:09.311+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:26:09.348+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:26:09.348+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:26:09.349+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:26:09.366+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:26:09.414+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:26:09.414+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:26:09.484+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:26:09.483+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:26:09.539+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.245 seconds
[2024-04-23T12:26:48.408+0000] {processor.py:161} INFO - Started process (PID=543) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T12:26:48.416+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T12:26:48.421+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:26:48.420+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T12:26:48.482+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T12:26:48.484+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:26:48.484+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T12:26:48.535+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T12:26:48.604+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:26:48.603+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T12:26:48.637+0000] {logging_mixin.py:188} INFO - [2024-04-23T12:26:48.637+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T12:26:48.763+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.381 seconds
[2024-04-23T17:37:41.589+0000] {processor.py:161} INFO - Started process (PID=559) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T17:37:41.648+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T17:37:41.685+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:37:41.685+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T17:37:43.041+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T17:37:44.096+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:37:44.155+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:37:44.238+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T17:37:44.372+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:37:44.372+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T17:37:44.543+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:37:44.542+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T17:37:46.664+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 5.088 seconds
[2024-04-23T17:38:17.030+0000] {processor.py:161} INFO - Started process (PID=575) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T17:38:17.045+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T17:38:17.047+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:38:17.047+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T17:38:17.226+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T17:38:17.231+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:38:17.232+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:38:17.459+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T17:38:18.011+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:38:18.011+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T17:38:18.155+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:38:18.154+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T17:38:19.241+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 2.233 seconds
[2024-04-23T17:38:49.865+0000] {processor.py:161} INFO - Started process (PID=590) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T17:38:49.866+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T17:38:49.868+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:38:49.867+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T17:38:49.912+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T17:38:49.913+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:38:49.914+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:38:49.932+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T17:38:49.980+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:38:49.979+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T17:38:50.021+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:38:50.021+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T17:38:52.869+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 3.015 seconds
[2024-04-23T17:39:23.283+0000] {processor.py:161} INFO - Started process (PID=607) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T17:39:23.294+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T17:39:23.296+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:39:23.296+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T17:39:23.344+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T17:39:23.345+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:39:23.346+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:39:23.393+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T17:39:23.526+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:39:23.525+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T17:39:23.671+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:39:23.671+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T17:39:23.777+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.517 seconds
[2024-04-23T17:39:54.209+0000] {processor.py:161} INFO - Started process (PID=622) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T17:39:54.212+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T17:39:54.215+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:39:54.214+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T17:39:54.263+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T17:39:54.264+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:39:54.266+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:39:54.306+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T17:39:54.427+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:39:54.426+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T17:39:54.504+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:39:54.503+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T17:39:54.676+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.487 seconds
[2024-04-23T17:40:25.050+0000] {processor.py:161} INFO - Started process (PID=638) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T17:40:25.054+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T17:40:25.056+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:40:25.055+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T17:40:25.109+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T17:40:25.110+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:40:25.112+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:40:25.146+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T17:40:25.206+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:40:25.205+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T17:40:25.282+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:40:25.282+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T17:40:25.561+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.526 seconds
[2024-04-23T17:40:55.747+0000] {processor.py:161} INFO - Started process (PID=654) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T17:40:55.751+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T17:40:55.753+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:40:55.753+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T17:40:55.793+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T17:40:55.794+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:40:55.795+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:40:55.818+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T17:40:55.967+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:40:55.966+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T17:40:56.082+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:40:56.081+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T17:40:56.211+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.480 seconds
[2024-04-23T17:41:26.590+0000] {processor.py:161} INFO - Started process (PID=671) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T17:41:26.592+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T17:41:26.595+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:41:26.594+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T17:41:26.640+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T17:41:26.641+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:41:26.642+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:41:26.665+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T17:41:26.739+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:41:26.739+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T17:41:26.844+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:41:26.843+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T17:41:26.940+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.365 seconds
[2024-04-23T17:41:57.733+0000] {processor.py:161} INFO - Started process (PID=691) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T17:41:57.745+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T17:41:57.749+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:41:57.749+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T17:41:57.927+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T17:41:57.935+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:41:57.957+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:41:58.056+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T17:41:58.234+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:41:58.234+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T17:41:58.438+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:41:58.437+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T17:41:58.686+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.024 seconds
[2024-04-23T17:42:41.479+0000] {processor.py:161} INFO - Started process (PID=709) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T17:42:41.558+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T17:42:41.599+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:42:41.576+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T17:42:41.687+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T17:42:41.700+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:42:41.701+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:42:41.788+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T17:42:41.995+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:42:41.995+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T17:42:42.154+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:42:42.154+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T17:42:44.043+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 2.610 seconds
[2024-04-23T17:43:18.169+0000] {processor.py:161} INFO - Started process (PID=723) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T17:43:18.210+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T17:43:18.256+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:43:18.255+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T17:43:18.348+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T17:43:18.349+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:43:18.350+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:43:18.376+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T17:43:18.514+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:43:18.514+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T17:43:18.606+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:43:18.605+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T17:43:19.001+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.880 seconds
[2024-04-23T17:46:44.426+0000] {processor.py:161} INFO - Started process (PID=61) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T17:46:44.439+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T17:46:44.450+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:46:44.445+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T17:46:44.944+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T17:46:44.972+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:46:44.986+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:46:45.129+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T17:46:46.508+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:46:46.507+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T17:46:46.821+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:46:46.821+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T17:46:48.276+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 3.881 seconds
[2024-04-23T17:47:30.236+0000] {processor.py:161} INFO - Started process (PID=87) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T17:47:30.323+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T17:47:30.527+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:47:30.525+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T17:47:31.565+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T17:47:31.580+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:47:31.582+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:47:31.895+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T17:47:35.528+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:47:35.515+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T17:47:36.597+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:47:36.596+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T17:47:37.525+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 7.370 seconds
[2024-04-23T17:48:08.184+0000] {processor.py:161} INFO - Started process (PID=108) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T17:48:08.381+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T17:48:08.385+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:48:08.384+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T17:48:08.414+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T17:48:08.415+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:48:08.416+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:48:08.430+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T17:48:08.468+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:48:08.468+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T17:48:08.508+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:48:08.508+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T17:48:09.556+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.385 seconds
[2024-04-23T17:58:19.826+0000] {processor.py:161} INFO - Started process (PID=109) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T17:58:20.141+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T17:58:20.236+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:58:20.236+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T17:58:21.033+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T17:58:21.220+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:58:21.221+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:58:21.242+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T17:58:21.477+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:58:21.477+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T17:58:21.527+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:58:21.526+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T17:58:23.127+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 3.321 seconds
[2024-04-23T17:59:06.882+0000] {processor.py:161} INFO - Started process (PID=132) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T17:59:07.058+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T17:59:07.074+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:59:07.073+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T17:59:07.527+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T17:59:07.532+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:59:07.533+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:59:07.546+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T17:59:07.673+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:59:07.672+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T17:59:07.785+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:59:07.784+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T17:59:11.537+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 4.670 seconds
[2024-04-23T17:59:49.196+0000] {processor.py:161} INFO - Started process (PID=148) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T17:59:49.224+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T17:59:49.289+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:59:49.288+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T17:59:54.272+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T17:59:54.309+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:59:54.317+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T17:59:54.352+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T17:59:54.595+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:59:54.594+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T17:59:54.796+0000] {logging_mixin.py:188} INFO - [2024-04-23T17:59:54.796+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T17:59:55.502+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 6.341 seconds
[2024-04-23T18:00:28.390+0000] {processor.py:161} INFO - Started process (PID=164) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:00:28.894+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:00:28.897+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:00:28.896+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:00:29.299+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:00:29.300+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:00:29.301+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:00:29.328+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:00:29.521+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:00:29.520+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:00:29.650+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:00:29.649+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:00:31.703+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 3.334 seconds
[2024-04-23T18:01:11.441+0000] {processor.py:161} INFO - Started process (PID=188) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:01:11.449+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:01:11.473+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:01:11.472+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:01:12.143+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:01:12.192+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:01:12.212+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:01:12.249+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:01:12.471+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:01:12.471+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:01:12.657+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:01:12.657+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:01:12.803+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.423 seconds
[2024-04-23T18:01:55.500+0000] {processor.py:161} INFO - Started process (PID=203) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:01:55.504+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:01:55.509+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:01:55.508+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:01:55.565+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:01:55.566+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:01:55.573+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:01:55.583+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:01:56.750+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:01:56.750+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:01:56.920+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:01:56.920+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:01:57.924+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 2.498 seconds
[2024-04-23T18:02:44.048+0000] {processor.py:161} INFO - Started process (PID=225) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:02:44.063+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:02:44.079+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:02:44.079+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:02:44.983+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:02:44.992+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:02:44.994+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:02:45.016+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:02:45.415+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:02:45.414+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:02:45.705+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:02:45.705+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:02:45.912+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.914 seconds
[2024-04-23T18:03:17.452+0000] {processor.py:161} INFO - Started process (PID=239) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:03:17.463+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:03:17.465+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:03:17.464+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:03:17.537+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:03:17.538+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:03:17.541+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:03:17.559+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:03:17.711+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:03:17.711+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:03:17.829+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:03:17.828+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:03:17.943+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.508 seconds
[2024-04-23T18:03:52.564+0000] {processor.py:161} INFO - Started process (PID=255) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:03:52.567+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:03:52.570+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:03:52.570+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:03:53.068+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:03:53.078+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:03:53.079+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:03:53.087+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:03:53.189+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:03:53.189+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:03:53.231+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:03:53.230+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:03:54.572+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 2.030 seconds
[2024-04-23T18:04:28.347+0000] {processor.py:161} INFO - Started process (PID=271) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:04:28.349+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:04:28.363+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:04:28.362+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:04:28.414+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:04:28.415+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:04:28.426+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:04:28.443+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:04:28.530+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:04:28.530+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:04:28.575+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:04:28.575+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:04:29.979+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.644 seconds
[2024-04-23T18:05:08.199+0000] {processor.py:161} INFO - Started process (PID=296) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:05:08.203+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:05:08.206+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:05:08.205+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:05:08.571+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:05:08.574+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:05:08.581+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:05:08.586+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:05:08.627+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:05:08.627+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:05:08.659+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:05:08.659+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:05:09.189+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.023 seconds
[2024-04-23T18:05:57.278+0000] {processor.py:161} INFO - Started process (PID=311) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:05:57.316+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:05:57.378+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:05:57.377+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:05:59.222+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:05:59.229+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:05:59.230+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:05:59.315+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:05:59.705+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:05:59.705+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:05:59.831+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:05:59.831+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:06:00.369+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 3.120 seconds
[2024-04-23T18:06:38.450+0000] {processor.py:161} INFO - Started process (PID=334) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:06:38.451+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:06:38.454+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:06:38.453+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:06:38.584+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:06:38.586+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:06:38.591+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:06:38.615+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:06:38.758+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:06:38.757+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:06:38.890+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:06:38.890+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:06:38.994+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.565 seconds
[2024-04-23T18:07:16.459+0000] {processor.py:161} INFO - Started process (PID=350) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:07:16.465+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:07:16.473+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:07:16.472+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:07:17.130+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:07:17.165+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:07:17.183+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:07:17.195+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:07:17.408+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:07:17.407+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:07:17.582+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:07:17.572+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:07:28.584+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 12.182 seconds
[2024-04-23T18:07:59.207+0000] {processor.py:161} INFO - Started process (PID=364) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:07:59.208+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:07:59.210+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:07:59.209+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:07:59.324+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:07:59.325+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:07:59.326+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:07:59.337+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:07:59.420+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:07:59.419+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:07:59.496+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:07:59.496+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:07:59.554+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.373 seconds
[2024-04-23T18:08:30.443+0000] {processor.py:161} INFO - Started process (PID=379) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:08:30.445+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:08:30.447+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:08:30.446+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:08:30.487+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:08:30.488+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:08:30.489+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:08:30.500+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:08:30.541+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:08:30.541+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:08:30.585+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:08:30.584+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:08:30.751+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.319 seconds
[2024-04-23T18:09:01.320+0000] {processor.py:161} INFO - Started process (PID=395) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:09:01.325+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:09:01.329+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:09:01.328+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:09:01.696+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:09:01.697+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:09:01.699+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:09:01.726+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:09:01.838+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:09:01.837+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:09:01.964+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:09:01.963+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:09:05.705+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 4.412 seconds
[2024-04-23T18:09:36.221+0000] {processor.py:161} INFO - Started process (PID=411) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:09:36.233+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:09:36.236+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:09:36.235+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:09:36.338+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:09:36.361+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:09:36.381+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:09:36.423+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:09:36.606+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:09:36.593+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:09:36.782+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:09:36.781+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:09:36.971+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.773 seconds
[2024-04-23T18:10:07.770+0000] {processor.py:161} INFO - Started process (PID=427) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:10:07.778+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:10:07.803+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:10:07.802+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:10:07.893+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:10:07.894+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:10:07.895+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:10:07.902+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:10:07.964+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:10:07.963+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:10:08.062+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:10:08.062+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:10:08.308+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.562 seconds
[2024-04-23T18:10:39.312+0000] {processor.py:161} INFO - Started process (PID=450) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:10:39.318+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:10:39.320+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:10:39.319+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:10:39.384+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:10:39.388+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:10:39.405+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:10:39.441+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:10:39.577+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:10:39.576+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:10:39.624+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:10:39.624+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:10:41.700+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 2.411 seconds
[2024-04-23T18:11:12.626+0000] {processor.py:161} INFO - Started process (PID=466) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:11:12.628+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:11:12.639+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:11:12.629+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:11:12.718+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:11:12.719+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:11:12.720+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:11:12.735+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:11:12.795+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:11:12.793+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:11:12.843+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:11:12.843+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:11:13.593+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.994 seconds
[2024-04-23T18:11:44.416+0000] {processor.py:161} INFO - Started process (PID=480) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:11:44.424+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:11:44.440+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:11:44.439+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:11:44.589+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:11:44.600+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:11:44.601+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:11:44.608+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:11:44.689+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:11:44.688+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:11:44.793+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:11:44.781+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:11:44.941+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.547 seconds
[2024-04-23T18:12:15.522+0000] {processor.py:161} INFO - Started process (PID=495) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:12:15.534+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:12:15.538+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:12:15.537+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:12:15.617+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:12:15.618+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:12:15.619+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:12:15.629+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:12:15.691+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:12:15.691+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:12:15.768+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:12:15.767+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:12:18.298+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 2.810 seconds
[2024-04-23T18:12:48.795+0000] {processor.py:161} INFO - Started process (PID=511) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:12:48.802+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:12:48.806+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:12:48.805+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:12:48.899+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:12:48.905+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:12:48.906+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:12:48.918+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:12:49.053+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:12:49.052+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:12:49.127+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:12:49.126+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:12:50.378+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.597 seconds
[2024-04-23T18:13:21.118+0000] {processor.py:161} INFO - Started process (PID=526) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:13:21.131+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:13:21.134+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:13:21.133+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:13:21.217+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:13:21.218+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:13:21.219+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:13:21.235+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:13:21.349+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:13:21.349+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:13:21.453+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:13:21.452+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:13:21.550+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.465 seconds
[2024-04-23T18:13:52.231+0000] {processor.py:161} INFO - Started process (PID=541) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:13:52.257+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:13:52.274+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:13:52.274+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:13:52.627+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:13:52.638+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:13:52.651+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:13:52.678+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:13:52.783+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:13:52.782+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:13:52.848+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:13:52.848+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:13:53.192+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.972 seconds
[2024-04-23T18:14:24.697+0000] {processor.py:161} INFO - Started process (PID=557) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:14:24.731+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:14:24.745+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:14:24.744+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:14:24.876+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:14:24.886+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:14:24.893+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:14:27.516+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:14:29.060+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:14:29.059+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:14:30.712+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:14:30.711+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:14:32.439+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 8.110 seconds
[2024-04-23T18:15:02.625+0000] {processor.py:161} INFO - Started process (PID=571) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:15:02.626+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:15:02.628+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:15:02.627+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:15:03.221+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:15:03.964+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:15:03.965+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:15:03.970+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:15:04.014+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:15:04.013+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:15:04.042+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:15:04.042+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:15:05.526+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 2.911 seconds
[2024-04-23T18:15:35.849+0000] {processor.py:161} INFO - Started process (PID=587) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:15:35.897+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:15:35.912+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:15:35.911+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:15:36.445+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:15:36.459+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:15:36.468+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:15:36.492+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:15:36.653+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:15:36.652+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:15:36.837+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:15:36.836+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:15:37.328+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.498 seconds
[2024-04-23T18:16:07.498+0000] {processor.py:161} INFO - Started process (PID=602) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:16:07.503+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:16:07.509+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:16:07.508+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:16:07.566+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:16:07.567+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:16:07.568+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:16:07.577+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:16:07.636+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:16:07.635+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:16:07.737+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:16:07.736+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:16:07.853+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.377 seconds
[2024-04-23T18:16:39.364+0000] {processor.py:161} INFO - Started process (PID=619) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:16:39.379+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:16:39.382+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:16:39.381+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:16:39.525+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:16:39.537+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:16:39.538+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:16:39.555+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:16:39.674+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:16:39.673+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:16:39.782+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:16:39.773+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:16:41.320+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.997 seconds
[2024-04-23T18:17:11.661+0000] {processor.py:161} INFO - Started process (PID=635) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:17:11.663+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:17:11.678+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:17:11.677+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:17:11.778+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:17:11.779+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:17:11.779+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:17:11.791+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:17:11.913+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:17:11.912+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:17:11.999+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:17:11.999+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:17:12.648+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.011 seconds
[2024-04-23T18:17:43.377+0000] {processor.py:161} INFO - Started process (PID=650) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:17:43.395+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:17:43.400+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:17:43.399+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:17:43.463+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:17:43.464+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:17:43.465+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:17:43.473+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:17:43.595+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:17:43.594+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:17:43.738+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:17:43.738+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:17:44.449+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.106 seconds
[2024-04-23T18:18:18.451+0000] {processor.py:161} INFO - Started process (PID=666) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:18:18.467+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:18:18.553+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:18:18.553+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:18:19.554+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:18:19.559+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:18:19.584+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:18:19.593+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:18:19.723+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:18:19.722+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:18:19.934+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:18:19.934+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:18:23.251+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 4.844 seconds
[2024-04-23T18:18:53.505+0000] {processor.py:161} INFO - Started process (PID=681) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:18:53.506+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:18:53.508+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:18:53.508+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:18:53.557+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:18:53.558+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:18:53.558+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:18:53.570+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:18:53.711+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:18:53.711+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:18:53.839+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:18:53.827+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:18:53.919+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.438 seconds
[2024-04-23T18:19:27.335+0000] {processor.py:161} INFO - Started process (PID=703) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:19:27.339+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:19:27.348+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:19:27.347+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:19:27.390+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:19:27.391+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:19:27.402+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:19:27.419+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:19:27.559+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:19:27.559+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:19:27.695+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:19:27.695+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:19:27.891+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.587 seconds
[2024-04-23T18:20:04.680+0000] {processor.py:161} INFO - Started process (PID=720) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:20:04.694+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:20:04.698+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:20:04.697+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:20:04.758+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:20:04.769+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:20:04.784+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:20:04.804+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:20:04.922+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:20:04.921+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:20:05.064+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:20:05.064+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:20:05.146+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.488 seconds
[2024-04-23T18:20:47.622+0000] {processor.py:161} INFO - Started process (PID=735) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:20:47.828+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:20:47.990+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:20:47.989+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:20:49.949+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:20:50.001+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:20:50.082+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:20:50.173+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:20:50.352+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:20:50.352+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:20:50.494+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:20:50.494+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:20:53.251+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 5.662 seconds
[2024-04-23T18:21:23.724+0000] {processor.py:161} INFO - Started process (PID=751) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:21:23.726+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:21:23.731+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:21:23.727+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:21:23.847+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:21:23.850+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:21:23.851+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:21:23.864+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:21:24.141+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:21:24.132+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:21:24.233+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:21:24.233+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:21:24.271+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.569 seconds
[2024-04-23T18:21:58.062+0000] {processor.py:161} INFO - Started process (PID=767) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:21:58.069+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:21:58.075+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:21:58.074+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:21:58.323+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:21:58.351+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:21:58.361+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:21:58.412+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:21:58.549+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:21:58.549+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:21:58.662+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:21:58.662+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:21:59.205+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.164 seconds
[2024-04-23T18:22:33.826+0000] {processor.py:161} INFO - Started process (PID=790) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:22:33.832+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:22:33.840+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:22:33.840+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:22:35.378+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:22:35.379+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:22:35.380+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:22:35.398+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:22:36.986+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:22:36.986+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:22:37.754+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:22:37.754+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:22:40.792+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 6.988 seconds
[2024-04-23T18:23:13.978+0000] {processor.py:161} INFO - Started process (PID=806) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:23:14.086+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:23:14.241+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:23:14.240+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:23:16.119+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:23:16.156+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:23:16.173+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:23:16.196+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:23:16.356+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:23:16.355+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:23:16.473+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:23:16.473+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:23:20.215+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 6.904 seconds
[2024-04-23T18:23:51.750+0000] {processor.py:161} INFO - Started process (PID=821) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:23:51.767+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:23:51.775+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:23:51.774+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:23:52.148+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:23:52.160+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:23:52.176+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:23:52.241+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:23:52.436+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:23:52.435+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:23:52.667+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:23:52.667+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:23:52.817+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.090 seconds
[2024-04-23T18:24:23.088+0000] {processor.py:161} INFO - Started process (PID=836) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:24:23.118+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:24:23.126+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:24:23.125+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:24:23.289+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:24:23.290+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:24:23.290+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:24:23.307+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:24:23.444+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:24:23.444+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:24:23.559+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:24:23.558+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:24:23.636+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.582 seconds
[2024-04-23T18:24:54.604+0000] {processor.py:161} INFO - Started process (PID=853) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:24:54.609+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:24:54.644+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:24:54.643+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:24:55.061+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:24:55.067+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:24:55.073+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:24:55.081+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:24:55.190+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:24:55.189+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:24:55.321+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:24:55.320+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:24:56.840+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 2.269 seconds
[2024-04-23T18:25:28.311+0000] {processor.py:161} INFO - Started process (PID=870) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:25:28.347+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:25:28.366+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:25:28.365+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:25:28.691+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:25:28.705+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:25:28.723+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:25:28.749+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:25:28.863+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:25:28.863+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:25:28.967+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:25:28.959+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:25:29.112+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.840 seconds
[2024-04-23T18:25:59.938+0000] {processor.py:161} INFO - Started process (PID=891) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:25:59.958+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:25:59.962+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:25:59.962+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:26:00.741+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:26:00.771+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:26:00.798+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:26:00.849+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:26:01.064+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:26:01.063+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:26:01.261+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:26:01.251+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:26:01.393+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.497 seconds
[2024-04-23T18:26:32.013+0000] {processor.py:161} INFO - Started process (PID=907) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:26:32.049+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:26:32.093+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:26:32.093+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:26:32.212+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:26:32.224+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:26:32.228+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:26:32.246+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:26:32.328+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:26:32.327+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:26:32.390+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:26:32.390+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:26:32.530+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.550 seconds
[2024-04-23T18:27:05.665+0000] {processor.py:161} INFO - Started process (PID=923) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:27:05.677+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:27:05.684+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:27:05.683+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:27:05.768+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:27:05.770+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:27:05.779+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:27:05.802+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:27:05.947+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:27:05.947+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:27:06.061+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:27:06.060+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:27:07.643+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 2.000 seconds
[2024-04-23T18:27:39.783+0000] {processor.py:161} INFO - Started process (PID=939) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:27:39.946+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:27:39.949+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:27:39.948+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:27:40.121+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:27:40.122+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:27:40.123+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:27:40.135+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:27:40.186+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:27:40.185+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:27:40.236+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:27:40.236+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:27:41.806+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 2.049 seconds
[2024-04-23T18:28:12.665+0000] {processor.py:161} INFO - Started process (PID=954) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:28:12.680+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:28:12.682+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:28:12.682+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:28:12.845+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:28:12.859+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:28:12.879+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:28:12.910+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:28:13.046+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:28:13.045+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:28:13.198+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:28:13.198+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:28:13.320+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.684 seconds
[2024-04-23T18:28:49.672+0000] {processor.py:161} INFO - Started process (PID=970) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:28:49.674+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:28:49.687+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:28:49.686+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:28:49.769+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:28:49.771+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:28:49.783+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:28:49.802+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:28:50.075+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:28:50.075+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:28:50.307+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:28:50.307+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:28:50.451+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.829 seconds
[2024-04-23T18:29:21.126+0000] {processor.py:161} INFO - Started process (PID=986) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:29:21.136+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:29:21.146+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:29:21.145+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:29:21.313+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:29:21.321+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:29:21.322+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:29:21.346+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:29:21.492+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:29:21.492+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:29:21.581+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:29:21.580+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:29:21.663+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.552 seconds
[2024-04-23T18:29:52.581+0000] {processor.py:161} INFO - Started process (PID=1003) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:29:52.594+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:29:52.599+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:29:52.598+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:29:52.686+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:29:52.689+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:29:52.690+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:29:52.697+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:29:52.787+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:29:52.786+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:29:52.846+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:29:52.846+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:29:52.898+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.350 seconds
[2024-04-23T18:30:23.016+0000] {processor.py:161} INFO - Started process (PID=1017) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:30:23.019+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:30:23.021+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:30:23.020+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:30:23.094+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:30:23.095+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:30:23.100+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:30:23.108+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:30:23.187+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:30:23.187+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:30:23.249+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:30:23.248+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:30:23.576+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.576 seconds
[2024-04-23T18:30:53.798+0000] {processor.py:161} INFO - Started process (PID=1033) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:30:53.799+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:30:53.800+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:30:53.800+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:30:53.824+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:30:53.825+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:30:53.825+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:30:53.830+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:30:53.870+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:30:53.869+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:30:53.922+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:30:53.921+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:30:53.994+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.203 seconds
[2024-04-23T18:31:28.334+0000] {processor.py:161} INFO - Started process (PID=1049) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:31:28.346+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:31:28.514+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:31:28.514+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:31:31.265+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:31:31.267+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:31:31.271+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:31:31.286+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:31:31.446+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:31:31.437+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:31:31.563+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:31:31.563+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:31:31.835+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 3.520 seconds
[2024-04-23T18:32:02.424+0000] {processor.py:161} INFO - Started process (PID=1064) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:32:02.426+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:32:02.428+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:32:02.427+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:32:02.454+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:32:02.455+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:32:02.455+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:32:02.466+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:32:02.533+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:32:02.532+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:32:02.580+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:32:02.579+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:32:02.633+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.219 seconds
[2024-04-23T18:32:38.549+0000] {processor.py:161} INFO - Started process (PID=1085) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:32:38.756+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:32:38.758+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:32:38.757+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:32:39.303+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:32:39.315+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:32:39.317+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:32:39.373+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:32:39.519+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:32:39.519+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:32:39.635+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:32:39.635+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:32:41.277+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 2.764 seconds
[2024-04-23T18:33:12.319+0000] {processor.py:161} INFO - Started process (PID=1101) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:33:12.333+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:33:12.337+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:33:12.336+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:33:12.642+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:33:12.643+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:33:12.645+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:33:12.673+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:33:12.817+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:33:12.816+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:33:12.945+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:33:12.944+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:33:19.176+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 6.879 seconds
[2024-04-23T18:33:51.933+0000] {processor.py:161} INFO - Started process (PID=1119) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:33:51.937+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:33:51.939+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:33:51.939+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:33:51.978+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:33:51.980+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:33:51.981+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:33:52.010+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:33:52.114+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:33:52.114+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:33:52.196+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:33:52.196+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:33:53.785+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.875 seconds
[2024-04-23T18:34:33.007+0000] {processor.py:161} INFO - Started process (PID=1135) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:34:33.025+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:34:33.028+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:34:33.028+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:34:33.342+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:34:33.345+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:34:33.346+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:34:33.361+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:34:33.504+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:34:33.504+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:34:33.622+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:34:33.621+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:34:33.728+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.757 seconds
[2024-04-23T18:35:43.068+0000] {processor.py:161} INFO - Started process (PID=1162) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:35:43.075+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:35:43.079+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:35:43.078+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:35:43.135+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:35:43.152+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:35:43.157+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:35:43.165+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:35:43.292+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:35:43.292+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:35:43.343+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:35:43.343+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:35:45.492+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 2.438 seconds
[2024-04-23T18:36:16.308+0000] {processor.py:161} INFO - Started process (PID=1176) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:36:16.316+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:36:16.323+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:36:16.320+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:36:16.558+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:36:16.576+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:36:16.584+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:36:16.611+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:36:16.761+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:36:16.760+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:36:17.005+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:36:17.005+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:36:17.991+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.708 seconds
[2024-04-23T18:36:56.245+0000] {processor.py:161} INFO - Started process (PID=1191) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:36:56.260+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:36:56.279+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:36:56.279+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:36:56.381+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:36:56.393+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:36:56.396+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:36:56.416+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:36:56.557+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:36:56.557+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:36:56.813+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:36:56.812+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:36:57.138+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.930 seconds
[2024-04-23T18:37:31.036+0000] {processor.py:161} INFO - Started process (PID=1207) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:37:31.049+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:37:31.051+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:37:31.050+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:37:31.115+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:37:31.116+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:37:31.128+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:37:31.152+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:37:31.335+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:37:31.323+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:37:31.562+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:37:31.562+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:37:31.797+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.782 seconds
[2024-04-23T18:38:03.613+0000] {processor.py:161} INFO - Started process (PID=1224) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:38:03.615+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:38:03.631+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:38:03.630+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:38:03.726+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:38:03.735+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:38:03.737+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:38:03.754+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:38:03.870+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:38:03.870+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:38:03.977+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:38:03.977+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:38:04.120+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.531 seconds
[2024-04-23T18:38:36.812+0000] {processor.py:161} INFO - Started process (PID=1240) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:38:36.814+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:38:36.816+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:38:36.815+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:38:36.848+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:38:36.849+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:38:36.850+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:38:36.859+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:38:36.902+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:38:36.901+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:38:36.940+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:38:36.939+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:38:37.763+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.967 seconds
[2024-04-23T18:39:08.141+0000] {processor.py:161} INFO - Started process (PID=1257) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:39:08.144+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:39:08.147+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:39:08.146+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:39:08.214+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:39:08.215+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:39:08.216+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:39:08.224+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:39:08.298+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:39:08.298+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:39:08.503+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:39:08.502+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:39:08.615+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.494 seconds
[2024-04-23T18:39:43.198+0000] {processor.py:161} INFO - Started process (PID=1273) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:39:43.205+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:39:43.208+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:39:43.207+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:39:43.274+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:39:43.278+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:39:43.280+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:39:43.290+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:39:43.396+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:39:43.395+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:39:43.459+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:39:43.458+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:39:44.181+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.003 seconds
[2024-04-23T18:40:15.956+0000] {processor.py:161} INFO - Started process (PID=1288) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:40:15.959+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:40:15.962+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:40:15.961+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:40:16.017+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:40:16.018+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:40:16.019+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:40:16.042+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:40:16.104+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:40:16.103+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:40:16.309+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:40:16.308+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:40:16.383+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 0.450 seconds
[2024-04-23T18:40:47.283+0000] {processor.py:161} INFO - Started process (PID=1307) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T18:40:47.285+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T18:40:47.301+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:40:47.300+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T18:40:47.495+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T18:40:47.497+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:40:47.497+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T18:40:47.513+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T18:40:47.631+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:40:47.631+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T18:40:47.749+0000] {logging_mixin.py:188} INFO - [2024-04-23T18:40:47.749+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T18:40:49.798+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 2.543 seconds
[2024-04-23T22:40:12.732+0000] {processor.py:161} INFO - Started process (PID=1340) to work on /opt/airflow/dags/Prueba.py
[2024-04-23T22:40:12.746+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/Prueba.py for tasks to queue
[2024-04-23T22:40:12.768+0000] {logging_mixin.py:188} INFO - [2024-04-23T22:40:12.767+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/Prueba.py
[2024-04-23T22:40:13.450+0000] {logging_mixin.py:188} INFO - /opt/Prueba2.py
[2024-04-23T22:40:13.527+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T22:40:13.564+0000] {logging_mixin.py:188} INFO - Error al ejecutar el script: [Errno 2] No such file or directory: '/opt/Prueba2.py'
[2024-04-23T22:40:13.593+0000] {processor.py:840} INFO - DAG(s) 'm_dag_ejecut.' retrieved from /opt/airflow/dags/Prueba.py
[2024-04-23T22:40:13.722+0000] {logging_mixin.py:188} INFO - [2024-04-23T22:40:13.722+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-23T22:40:13.865+0000] {logging_mixin.py:188} INFO - [2024-04-23T22:40:13.865+0000] {dag.py:3954} INFO - Setting next_dagrun for m_dag_ejecut. to 2024-04-07 00:00:00+00:00, run_after=2024-04-08 00:00:00+00:00
[2024-04-23T22:40:14.277+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/Prueba.py took 1.567 seconds
